test/Feature/UpdateSpec.hs

```diff
@@ -91,23 +100,38 @@ spec = do
           }
 
       it "can update multiple items" $ do
-        replicateM_ 10 $ post "/auto_incrementing_pk"
-          [json| { non_nullable_string: "a" } |]
-        replicateM_ 10 $ post "/auto_incrementing_pk"
-          [json| { non_nullable_string: "b" } |]
```

Comment:
This one I don't understand. Where did the `replicateM_` tests went? Or are they not necessary now?

Reply:
Yeah, that's not easy from the diff, because you need context that is not in here.

This test is about `PATCH`ing multiple rows at the same time. The `replicateM_` are just calls to prepare the `auto_incrementing_pk` to hold multiple rows with the same values. I wanted to avoid so many persistant changes like that, because they would also affect the serial pk column a lot, even if undone. So I went looking for another table where we already have a few rows with the same values in one column, to make a "multiple-rows-patch-request" against.

In `data.sql` we have:

```
INSERT INTO no_pk VALUES (NULL, NULL);
INSERT INTO no_pk VALUES ('1', '0');
INSERT INTO no_pk VALUES ('2', '0');
```

Ok, this test does not change 10 rows at the same time anymore, but only 2 - but that's the same.

The new test changes those `0` to `1`. So I test for 0 first, then make the patch request, then test for 1, then make the reverse patch request to put the data back in the original state.

Reply:
Aha, got it!